docs/source/conf.py

```diff
@@ -98,7 +98,7 @@
 html_theme_options = {
     "conf_py_path": "docs/source/",
     "hide_edit_this_page_button": "false",
-    "hide_banner": "true",
+    "hide_banner": "false",
```

Comment:
why updating the deafult value?

Reply:
For demo purposes. Added back the previous value.